packages/server/modules/auth/defaultApps.ts

```diff
@@ -150,3 +152,10 @@ export function getDefaultApps() {
 export function getDefaultApp({ id }: { id: string }) {
   return defaultApps.find((app) => app.id === id) || null
 }
+
+export type DefaultApp = (typeof defaultApps)[number]
```

Comment:
I didn't make getDefaultApps() injectable, because its ultimately a compile-time constant that stays static.